### PR TITLE
chore(): use deconstruction and constants in place of strings to save some bytes of code

### PR DESCRIPTION
**Pull request by asturur**
## Description
This is no functional changes a tentative to recover some bundle size

### CODE REVIEW

File: src/canvas/canvas_gestures.mixin.ts
Comment (ShaMan123) [inline src/canvas/canvas_gestures.mixin.ts:1]: What is the fate of this file?

File: src/constants.ts [hunk 1]
```diff
@@ -25,3 +25,15 @@ export const RIGHT = 'right';
...
 
 export const reNewline = /\r?\n/;
+
+export const MOVING = 'moving';
+export const SCALING = 'scaling';
+export const ROTATING = 'rotating';
+export const ROTATE = 'rotate';
+export const SKEWING = 'skewing';
+export const RESIZING = 'resizing';
+export const MODIFYPOLY = 'modifypoly';
```
Comment (ShaMan123) [inline src/constants.ts]: can we rename this?

File: src/controls/scaleSkew.ts [hunk 2]
```diff
@@ -26,11 +27,11 @@ export const scaleOrSkewActionName: ControlCallback<
   const isAlternative = isAltAction(eventData, fabricObject);
   if (control.x === 0) {
     // then is scaleY or skewX
-    return isAlternative ? 'skewX' : 'scaleY';
+    return isAlternative ? 'skewX' : SCALE_Y;
   }
   if (control.y === 0) {
     // then is scaleY or skewX
-    return isAlternative ? 'skewY' : 'scaleX';
```
Comment (ShaMan123) [inline src/controls/scaleSkew.ts]: skewX/Y?

**Review by ShaMan123 — APPROVED** [review]
In an ideal condition I would ask this to be merged once our queue is empty to avoid annoying merge conflicts with open PRs because it is a lot of diff ink

But we are not there yet so it is what it is

**Comment by asturur** [comment]
this can wait a lot is not important

File: src/canvas/canvas_gestures.mixin.ts
Comment (asturur) [reply src/canvas/canvas_gestures.mixin.ts:1]: at some point we need to find a solution for gestures

Comment (ShaMan123) [reply src/canvas/canvas_gestures.mixin.ts:1]: I gave some thought.
We can introduce a control that fits the entire object for a 2 finger gesture that scales and rotates the object.
The math sounds straight forward.
Measuring the diff of the vector created between the 2 touches.
Then the dev can do for 3 fingers whatever based on the 2 finger control.
Thoughts?

Comment (ShaMan123) [reply src/canvas/canvas_gestures.mixin.ts:1]: I would then argue that a mobile/touch app will not want the default controls

Comment (ShaMan123) [reply src/canvas/canvas_gestures.mixin.ts:1]: Then touch become a first class citizen in fabric

**Pull request merged**